rbac/management/role/definer.py

```diff
@@ -254,3 +257,34 @@ def delete_permission(permission: Permission):
             dual_write_handler.replicate_update_system_role()
         else:
             dual_write_handler.replicate_new_or_updated_role(role)
+
+
+def _seed_v2_role_from_v1(v1_role, display_name, description, public_tenant, v1_was_created):
+    """Create or update V2 role from V1 role during seeding."""
+    try:
+        v2_role, v2_created = SeededRoleV2.objects.get_or_create(
+            name=display_name,
+            tenant=public_tenant,
+            defaults={
+                "description": description,
+                "v1_source": v1_role,
+            },
+        )
```

Comment:
We will also need to set `v1_source` to `v1_role` and add the role to the `children` of the V2 appropriate platform role (... which we will also need to create during seeding).